includes/sql-schema/update.php

```diff
@@ -36,34 +36,6 @@
     $insert = 1;
 }
 
-// For transition from old system
```

Comment:
I literally can't find where we do anything with revision - I think it's a hang up from the fork so got rid of it.